mkdocs.yml

```diff
@@ -64,24 +64,30 @@ nav:
               - Base Definitions: reference/legacy/built-in/base.md
   - Metrics Glossary:
       - Metrics Glossary: metrics/index.md
-      - metrics/accuracy.md
-      - metrics/average-precision.md
-      - metrics/averaging-methods.md
-      - metrics/bertscore.md
-      - metrics/bleu.md
-      - metrics/confusion-matrix.md
-      - metrics/f1-score.md
-      - metrics/fpr.md
-      - metrics/geometry-matching.md
-      - metrics/iou.md
-      - metrics/meteor.md
-      - metrics/precision.md
-      - metrics/pr-curve.md
-      - metrics/recall.md
-      - metrics/roc-curve.md
-      - metrics/rouge-n.md
-      - metrics/specificity.md
-      - metrics/tp-fp-fn-tn.md
+      - Classification Metrics:
+        - Classification Metrics: metrics/classification/index.md
+        - metrics/accuracy.md
+        - metrics/average-precision.md
+        - metrics/averaging-methods.md
+        - metrics/confusion-matrix.md
+        - metrics/f1-score.md
+        - metrics/fpr.md
+        - metrics/precision.md
+        - metrics/pr-curve.md
+        - metrics/recall.md
+        - metrics/roc-curve.md
+        - metrics/specificity.md
+        - metrics/tp-fp-fn-tn.md
+      - NLP Metrics:
```

Comment:
Prefer the title "Natural Language Processing" instead of "NLP Metrics"

Reply:
I would remove the "Metrics" and just use "Natural Language Processing". Metrics are implied, given that this is all inside the "Metrics Glossary".

Reply:
Gotcha. All comments have been addressed now.